Add Audio and Sound actions

danielb987 opened the pull request:
Add LogixNG actions Audio and Sound.

dsand47 reviewed (commented):
The long description for the Sound action should indicate the sound name.

![Screen Shot 2021-06-15 at 12 23 04 AM](https://user-images.githubusercontent.com/17280992/121997578-325e3c80-cd70-11eb-8750-5d24d990e687.png)

danielb987 commented:
> The long description for the Sound action should indicate the sound name.

Fixed.

dsand47 reviewed (dismissed):
It really helps to have the sound file in the long description.

dsand47 commented on java/src/jmri/jmrit/logixng/actions/ActionLightIntensity.java:
```diff
@@ -0,0 +1,333 @@
+                            .evaluate();
+        }
+        
+        light.setTargetIntensity(intensity);
```
The valid range for intensity is 0.0 to 1.0.

danielb987 replied:
Fixed. Due to rounding errors, a formula might return -0.01 when 0.0 is expected, or 1.01 when 1.0 is expected. So the action now ensures that everything below 0.0 is set to 0.0 and everything above 1.0 is set to 1.0.

dsand47 reviewed (commented):
The long description should include the light name.
![Screen Shot 2021-06-16 at 2 23 20 PM](https://user-images.githubusercontent.com/17280992/122280224-9ab33800-ceae-11eb-8ebd-13e5573cef5d.png)

The text to too technical.
![Screen Shot 2021-06-16 at 2 14 46 PM](https://user-images.githubusercontent.com/17280992/122280330-b3bbe900-ceae-11eb-92ef-74e84cfb9dde.png)

Maybe something like `The light intensity, provided by Intensity analog expression child,`.

An alternative is not have the text and rely on the LogixNG reference entry.  

An unrelated issue when selecting `evaluate` for the Intensity line from the context menu:
```
     [java] 2021-06-16 14:17:38,931 ptionhandler.UncaughtExceptionHandler ERROR - Uncaught Exception caught by jmri.util.exceptionhandler.UncaughtExceptionHandler [AWT-EventQueue-0]
     [java] java.lang.RuntimeException: Not supported
     [java] 	at jmri.jmrit.logixng.expressions.swing.AbstractAnalogExpressionSwing.executeEvaluate(AbstractAnalogExpressionSwing.java:29)
     [java] 	at jmri.jmrit.logixng.tools.swing.TreeEditor.executeEvaluate(TreeEditor.java:1211)
     [java] 	at jmri.jmrit.logixng.tools.swing.TreeEditor$PopupMenu.actionPerformed(TreeEditor.java:1734)
     [java] 	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
     [java] 	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
```

danielb987 commented:
> The long description should include the light name.

I agree. Fixed.

> The text to too technical.

Yes. I have removed the technical part.

![DaveSand9](https://user-images.githubusercontent.com/20255317/122303392-c14f8e00-cf03-11eb-97c0-ed941a8fb112.png)

> An unrelated issue when selecting `evaluate` for the Intensity line from the context menu:

Thanks for spotting that. Fixed.

dsand47 reviewed (dismissed):
Looks good.
